Document VTTRegion instance properties (#44494)
mdn/content: files/en-us/web/api/vttregion/scroll/index.md

---
title: "VTTRegion: scroll property"
short-title: scroll
slug: Web/API/VTTRegion/scroll
page-type: web-api-instance-property
browser-compat: api.VTTRegion.scroll
---

{{APIRef("WebVTT")}}

> The **`scroll`** property of the {{domxref("VTTRegion")}} interface is an enumerated value representing how adding a new cue will move existing cues in the region.

Reviewer: ```suggestion
The **`scroll`** property of the {{domxref("VTTRegion")}} interface is an enumerated value indicating how existing cues in the region move when a new cue is added.
```

## Value

A string. Possible values are:

- `""` (the empty string)

>   - : The cues in the region are not to scroll, and are instead displayed simultaneously. This is the default.

Reviewer: ```suggestion
  - : Existing cues in the region don't scroll; instead, they are displayed at the same time as new cues. This is the default.
```

- `"up"`

>   - : The cues in the region are added at the bottom of the region and scroll any already-displayed cues upwards.

Reviewer: ```suggestion
  - : New cues are added at the bottom of the region, and existing cues scroll upwards to make space for them.
```

## Examples

> In the following example a new {{domxref("VTTRegion")}} is created, then the value of `scroll` is set to `"up"`. The value is then printed to the console.

Reviewer: ```suggestion
In the following example, a new {{domxref("VTTRegion")}} is created, then the value of `scroll` is set to `"up"`. The value is then printed to the console.
```
